nf_core/pipeline-template/conf/test.config

```diff
@@ -16,8 +16,8 @@ params {
 
     // Limit resources so that this can run on GitHub Actions
     max_cpus   = 2
-    max_memory = 6.GB
-    max_time   = 6.h
+    max_memory = '6.GB'
+    max_time   = '6.h'
```

Comment:
Why include the quotes? Even in the Nextflow documentation examples, quotes are not used: https://www.nextflow.io/docs/latest/config.html#process-selectors, I would keep it simple.

Reply:
I saw that this has been also updated in the `nextflow.config` template, though... 😓

Reply:
I think that this was to do with the JSON schema validation.